src/sql/workbench/contrib/executionPlan/browser/executionPlanView.ts

```diff
@@ -548,3 +558,18 @@ export class CompareExecutionPlanAction extends Action {
 		context.compareCurrentExecutionPlan();
 	}
 }
+
+export class TopOperationsAction extends Action {
+
+	public static ID = 'ep.topOperationsAction';
+	public static LABEL = localize('executionPlanTopOperationsAction', "Top Operations");
```

Comment:
We're starting to get a lot of options in the context menu - you should sync up with @erinstellato-ms and chat about whether it makes sense to try and at least split them up into sections so it isn't just one huge list of stuff